src/readit/profile/profile-public.js

```diff
@@ -0,0 +1,109 @@
+    if (vid === lid) {
+        navigate("/readit/profile");
+        return;
+    }
+    return(
+        <div>
+            <div className="row">
+                <div className="col-1">
+                    <i className="fs-4 bi bi-book"></i>
+                </div>
+                <div className="col-10">
+                    <b className="fs-3">{profile.firstName} {profile.lastName}</b>
+                    <div className="fs-6"><b>{profile.role}</b></div>
+                    <div className="wd-small-font wd-fg-color-lightgray">{profile.booksRead} Books Read {`${(profile.role === "CRITIC" || profile.role === "AUTHOR") ? `| ${profile.comments.length} Reviews Posted` : ''}`} {`${(profile.role === "AUTHOR") ? `| ${profile.numBooksWritten} Books Written` : ''}`} </div>
+                </div>
+            </div>
+            <img src={`/images/books.jpeg`} width="100%" height={150} className="mt-2" />
+            <div className="row mt-5">
+                <div className="col-8">
+                    <img src={profile.profilePicture} className="ms-4 wd-profile-img rounded-circle"/>
+                </div>
+                <span className="col-3 mt-2 ms-3">
+                    <button onClick={followHandler} className={`rounded-pill btn ${(following) ? "btn-dark" : "btn-outline-dark"} float-end fw-bold`}>{(following) ? "Following" : "Follow"}</button>
+                </span>
+            </div>
+            <div className="ms-4 mt-3">
+                <b className="fs-5">{profile.firstName} {profile.lastName}</b>
+                <div className="wd-small-font wd-fg-color-lightgray">@{profile.username}</div>
+            </div>
+            <div className="row ms-3 mt-2">
+                <div className="col-6 mt-4">
+                    <h5>Following ({currentUser.following.length})</h5>
+                    <ol>
+                        {currentUser.following.map(f => {
+                            const curr = users.find((u) => u._id === f);
+                            return (
+                                <li><Link to={`/readit/profile/${lid}/${f}`}>{curr.firstName} {curr.lastName}</Link></li>
+                            )
+                        }
+                        )}
+                        
+                    </ol>
+                </div>
+                <div className="col-6 mt-4">
+                    <h5>Followers ({currentUser.followers.length})</h5>
+                    <ol>
+                        {currentUser.followers.map(f => {
+                            const curr = users.find((u) => u._id === f);
+                            return (
+                                <li><Link to={`/readit/profile/${lid}/${f}`}>{curr.firstName} {curr.lastName}</Link></li>
+                            )}
+                            )}
+                    </ol>
+                </div>
+            </div>
+            {/* <div className="ms-3 mt-4" hidden={`${(profile.role === "VIEWER") ? 'hidden' : ''}`}>
```

Comment:
Should we remove commented out code?

Reply:
I left this in as a reminder to add reviews/comments that should come from the Details page. I can wait until then to merge this PR.